pkgs/development/python-modules/identify/default.nix

```
@@ -25,8 +25,11 @@ buildPythonPackage rec {
 
   build-system = [ setuptools ];
 
+  propagatedNativeBuildInputs = [ cffi ];
+
+  propagatedBuildInputs = [ cffi ];
+
   dependencies = [
```

Still should go to the existing attrs
```suggestion
  build-system = [
    setuptools
    cffi
  ];

  dependencies = [
    cffi
```